backend/drm/drm.c

```diff
@@ -1537,22 +1459,6 @@ static void drm_connector_cleanup(struct wlr_drm_connector *conn) {
 	switch (conn->state) {
 	case WLR_DRM_CONN_CONNECTED:
 	case WLR_DRM_CONN_CLEANUP:;
```

Comment:
```suggestion
	case WLR_DRM_CONN_CLEANUP:
```
semicolon not needed for fall through. Should remove?

Reply:
Pushed a fix